assets/src/components/customers/CustomerUtils.ts

```diff
@@ -0,0 +1,15 @@
+import * as API from '../../api';
+import logger from '../../logger';
+import {download} from '../../utils';
+
+export const exportCustomerData = async (customerId: string) => {
```

Comment:
Seemed like a fairly component-agnostic utility function so I extracted it into its own utils file.